test/widgets_test.rb

```diff
@@ -13,6 +13,7 @@ def stub_widget_value(id, value)
   before do
     allow(root_user).to receive(:root?).and_return(true)
     allow(subject).to receive(:root_user).and_return(root_user)
+    allow(subject).to receive(:user_to_validate).and_return(root_user)
```

Comment:
I think that test coverage is decreasing because `#user_to_validate` now has several lines of code and they are all mocked. Maybe we can simply mock `Users#new`.